### PR TITLE
Use seconds in time offset - Closes #205

### LINKED ISSUE #205: Use seconds instead of ms in time offset for transactions
To conform to Lisk Core standard.

### PR DESCRIPTION
**Pull request by willclarktech**
Merge #190 first
Closes #205

### CODE REVIEW

File: lib/time/slots.js [hunk 1]
```diff
@@ -69,6 +69,20 @@ function getTime (time) {
...
 /**
+* @method getTimeWithOffset
+* @param offset
+* @return {number}
+*/
+
+function getTimeWithOffset (offset) {
+	var now = new Date().getTime();
+	var time = offset
+		? now - (offset * 1000)
```
Comment (slaweet) [inline lib/time/slots.js]: I know we need to subtract the offset, but usually, offset is added. The API might be confusing. Maybe rather change minus to plus here and use negative offset.

File: test/transactions/signature.js [hunk 2]
```diff
@@ -35,34 +30,32 @@ describe('signature.js', function () {
...
+				(sgn).should.have.property('timestamp').and.be.equal(slots.getTime());
+			});
 
-		it('should use time slots with an offset to get the time for the timestamp', function () {
-			var now = new Date();
-			var clock = sinon.useFakeTimers(now, 'Date');
-			var offset = 10e3;
-			var time = 36174862;
-			var stub = sinon.stub(slots, 'getTime').returns(time);
+			it('should use time slots with an offset to get the time for the timestamp', function () {
```
Comment (Isabello) [inline test/transactions/signature.js]: specify how much of an offset in the description.

File: test/transactions/transaction.js [hunk 3]
```diff
@@ -31,35 +26,33 @@ describe('transaction.js', function () {
...
+				(trs).should.have.property('timestamp').and.be.equal(slots.getTime());
+			});
 
-		it('should use time slots with an offset to get the time for the timestamp', function () {
-			var now = new Date();
-			var clock = sinon.useFakeTimers(now, 'Date');
-			var offset = 10e3;
-			var time = 36174862;
-			var stub = sinon.stub(slots, 'getTime').returns(time);
+			it('should use time slots with an offset to get the time for the timestamp', function () {
```
Comment (Isabello) [inline test/transactions/transaction.js]: as previous

File: test/transactions/transfer.js [hunk 4]
```diff
@@ -77,35 +74,32 @@ describe('transfer.js', function () {
...
-			var now = new Date();
-			var clock = sinon.useFakeTimers(now, 'Date');
-			var offset = 10e3;
-			var time = 36174862;
-			var stub = sinon.stub(slots, 'getTime').returns(time);
+				(trs).should.have.property('timestamp').and.be.equal(slots.getTime());
+			});
 
-			var trs = createInTransfer(dappId, amount, secret, null, offset);
+			it('should use time slots with an offset to get the time for the timestamp', function () {
```
Comment (Isabello) [inline test/transactions/transfer.js]: as previous...

File: test/transactions/vote.js [hunk 5]
```diff
@@ -85,10 +77,10 @@ describe('vote.js', function () {
 			});
 
 			it('should have recipientId string equal to sender', function () {
-				(vt).should.have.property('recipientId').and.be.type('string').and.equal(lisk.crypto.getAddress(publicKey));
+				(vt).should.have.property('recipientId').and.be.type('string').and.equal(cryptoModule.getAddress(publicKey));
 			});
 
-			it('should have amount number eaul to 0', function () {
+			it('should have amount number equal to 0', function () {
```
Comment (Isabello) [inline test/transactions/vote.js]: :)

File: lib/time/slots.js (hunk 1 above)
Comment (Isabello) [inline lib/time/slots.js]: agreeing with this comment

File: test/transactions/signature.js (hunk 2 above)
Comment (willclarktech) [inline test/transactions/signature.js]: Why? It's not part of the specification.

**Pull request merged**